core/src/test/resources/HistoryServerExpectations/one_app_multi_attempt_json_expectation.json

```diff
@@ -11,6 +14,9 @@
     "completed" : true
   }, {
     "attemptId" : "1",
+    "startTimeEpoch" : 1426533910242,
+    "endTimeEpoch" : 1426533945177,
+    "lastUpdatedEpoch" : "",
```

Comment:
I think the issue is that `lastUpdated` is a formatting string, so an empty string makes sense for no value, but not for a numeric value?